en/docs/monitoring-and-insights/logs/view-pdp-logs.md

```diff
@@ -0,0 +1,73 @@
+# View Private Data Plane (PDP) Logs 
+
+Choreo offers the capability to access runtime logs through its console. However, in cases where viewing logs for your PDP is not supported by Choreo, you can still view the runtime logs of your components via the log analyzing engines associated with your PDP.
+
+## Prerequisites
+
+Before you try out this guide, complete the following:
```

Comment:
please complete the following steps:

Reply:
We avoid using please in the documentation generally.